src/components/Question/Question.js

```diff
@@ -257,7 +258,7 @@ export class Question extends Component {
                   >
                     Hide Explanation
                   </button>
-                  <div className="explanation">
+                  <div className="explanation" style={{ lineHeight: '2' }}>
```

Comment:
I know this is an issue spread across the app as of right now, but do you think you could move your inline style declarations to the correct styled component (`form` nested inside `Wrapper`, for this particular comment)? For scalability purposes I think it's a good idea to avoid inlining styles, especially since we have a redesign in progress!